src/app/services/courses.service.spec.ts

```diff
@@ -13,4 +14,11 @@ describe('CoursesService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  it('should load courses list', (done: DoneFn) => {
```

Comment:
you can use `async` helper from `@angular/testing`  as follows:

```typescript
it('should load courses list', async(() => { ... }));
```

Reply:
Ok